parsec/core/fs/workspacefs/file_operations.py

```diff
@@ -2,7 +2,18 @@
 from __future__ import annotations
 
 # Imports
-from parsec._parsec import prepare_read, prepare_reshape, prepare_resize, prepare_write
+from parsec._parsec import (
+    Chunk,
+    ChunkID,
+    prepare_read,
+    prepare_reshape,
+    prepare_resize,
+    prepare_write,
+)
+
+Chunks = tuple[Chunk, ...]
+ChunkIDSet = set[ChunkID]
```

Comment:
Are those used somewhere? I'm not a big fan of the alias on simple things like lists, it makes it less explicit. If we take something like `Chunks` and I see this type in the codebase, now I have to wonder weither it's a class on its own or a collection, and what type of collection.

Reply:
Those alias aren't used, I've removed them